src/expression.c

```diff
@@ -828,6 +946,24 @@ Expression *resolveUFCS(Scope *sc, CallExp *ce)
                 else
                     return NULL;
             }
+            if (t->ty == Tclass || t->ty == Tstruct)
+            {
+                Expressions results;
+                iterateAliasThis(sc, eleft, &atSubstUnaDotId, (void*)ce, &results);
+                if (results.dim == 1)
+                {
+                    return results[0];
+                }
+                else if (results.dim > 1)
+                {
+                    ce->error("Unable to unambiguously resolve %s Candidates:", ce->toChars());
+                    for (size_t j = 0; j < results.dim; ++j)
+                    {
+                        ce->error("%s", results[j]->toChars());
```

Comment:
I'd rather the use of ``errorSupplemental`` here, that way the list of conflicting symbols is correctly indented.